src/raps/actions/ens.ts

```diff
@@ -673,7 +675,11 @@ const executeAction = async (
   try {
     logger.log('[2 INNER] executing type', type);
     const actionPromise = findENSActionByType(type);
-    nonce = await actionPromise(wallet, rap, index, parameters as RapENSActionParameters, baseNonce);
+    nonce = await performanceTracking.getState().executeFn({
+      fn: actionPromise,
```

Comment:
this will take a different path during an error: the actions from `findENSActionByType` can throw errors, which will get swallowed by the `executeFn` and return `undefined` instead of throwing and ending in the catch